index.php

```diff
@@ -4,7 +4,7 @@
 
 use TopCat\Hydrators\CatHydrator;
 
-$db = new DB();
+$db = new TopCat\Utilities\DB();
 $dbconnection = $db->dbConnect();
```

Comment:
$dbconnection should be camelcased for consistency.

Reply:
not in this task. see s3t6-cleanup